kuma/users/newsletter/checks.py

```diff
@@ -0,0 +1,36 @@
+from django.conf import settings
+from django.core.checks import Error
+
+from . import sendinblue
+
+SENDINBLUE_ERROR = "kuma.users.sendinblue.E001"
+
+
+def sendinblue_check(app_configs, **kwargs):
+    if not settings.SENDINBLUE_API_KEY:
+        return []
+
+    response = sendinblue.request("GET", "contacts/attributes")
+    if not response.ok:
+        return [
+            Error(
+                f"Error when creating sendinblue attribute: {response.json()['message']}",
```

Comment:
That string isn't really right. Also, I don't think you can assume that it's JSON. 
```suggestion
                f"Error getting sendinblue attributes: {response.status_code}",
```